R/designOfExperiments.R

```diff
@@ -17,4 +17,188 @@
 
 designOfExperiments <- function(jaspResults, dataset, options, ...){
 
+  .qualityControlShowAvailableDesigns(options, jaspResults, position = 1)
+
+  .qualityControlSummarySelectedDesign(options, jaspResults, position = 2)
+
+  .qualityControlShowSelectedDesign(options, jaspResults, position = 3)
+
+}
+
+.qualityControlShowAvailableDesigns <- function(options, jaspResults, position){
+
+  if(is.null(jaspResults[["displayDesigns"]])){
+
+    tableTitle <- base::switch(options[["design"]],
+                               "factorial" = gettext("Available Factorial Designs (with Resolution)"),
+                               "screening" = gettext("Available Screening Designs"),
+                               "response" = gettext("Available Response Surface Designs"),
+                               "mixture" = gettext("Available Mixture Designs"),
+                               "taguchi" = gettext("Available Taguchi Designs"))
+
+    table <- createJaspTable(tableTitle)
+    table$position <- position
+
+    table$dependOn(options = "design")
+
+    if(options[["design"]] == "factorial"){
+
+      table$addColumnInfo(name = 'run', 		title = gettext("Runs"), 	type = 'integer')
```

Comment:
I don't care too much about mixing tabs and spaces in R files, but everywhere else we align with spaces in R files.
```suggestion
      table$addColumnInfo(name = 'run',         title = gettext("Runs"),         type = 'integer')
```
Either way, I suggest changing the tabs to spaces. If you want to stick to tabs (also fine by me) I'd remove the space after the `,`, which is redundant.